Source/BindableFunctions/Misc.cs

```diff
@@ -80,6 +83,28 @@ public static void ForcePause()
             }
         }
 
+        [BindableMethod(name = "Reset Encounters", category = "Misc")]
+        public static void ResetProxyFSMEncounters()
+        {
+            try
+            {
+                //literally couldnt figure out how to get this to not be awful to look at
+                //this resets the fsm responsible for a couple weird persistent values with the knight
+                GameObject knight = GameObject.Find("Knight");
+                PlayMakerFSM proxyFSM = knight.LocateMyFSM("ProxyFSM");
+                proxyFSM.FsmVariables.FindFsmBool("Faced Radiance").Value = false;
+                proxyFSM.FsmVariables.FindFsmBool("Faced Nightmare").Value = false;
+                proxyFSM.FsmVariables.FindFsmBool("Faced Zote").Value = false;
+
+
+            }
+            catch (Exception e)
+            {
```

Comment:
I don't think it's necessary to try catch them. If those fsm variables or the knight don't exist when this is run there's a bigger problem lmao

Reply:
I just threw a catcher there because the one above it did didn't think about it, you can get rid of it